src/20231113_Multi_IncreaseStablecoinOptimalBorrowRates/AaveV3Arbitrum_IncreaseStablecoinOptimalBorrowRates_20231113.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3ArbitrumAssets} from 'aave-address-book/AaveV3Arbitrum.sol';
+import {AaveV3PayloadArbitrum} from 'aave-helpers/v3-config-engine/AaveV3PayloadArbitrum.sol';
+import {EngineFlags} from 'aave-helpers/v3-config-engine/EngineFlags.sol';
+import {IAaveV3ConfigEngine} from 'aave-helpers/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {IV3RateStrategyFactory} from 'aave-helpers/v3-config-engine/IV3RateStrategyFactory.sol';
+
+/**
+ * @title Increase Stablecoin Optimal Borrow Rates
+ * @author Alice Rozengarden (@Rozengarden - Aave-chan initiative)
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x914862039828294f4277ad63087ffae295b7693ba365c9036326cca802bfc7af, 0xb9b28f57f7633dd6b987de9abcede23da62fe4fab6b002f189b8b25a7c02ea93
+ * - Discussion: https://governance.aave.com/t/arfc-increase-optimal-borrow-rates-for-ethereum-stablecoin-markets/15096/3
+ */
+contract AaveV3Arbitrum_IncreaseStablecoinOptimalBorrowRates_20231113 is AaveV3PayloadArbitrum {
+  function rateStrategiesUpdates()
+    public
+    pure
+    override
+    returns (IAaveV3ConfigEngine.RateStrategyUpdate[] memory)
+  {
+    IAaveV3ConfigEngine.RateStrategyUpdate[]
+      memory rateStrategies = new IAaveV3ConfigEngine.RateStrategyUpdate[](7);
+    rateStrategies[0] = IAaveV3ConfigEngine.RateStrategyUpdate({
+      asset: AaveV3ArbitrumAssets.DAI_UNDERLYING,
+      params: IV3RateStrategyFactory.RateStrategyParams({
+        optimalUsageRatio: _bpsToRay(90_00),
+        baseVariableBorrowRate: EngineFlags.KEEP_CURRENT,
+        variableRateSlope1: _bpsToRay(5_00),
+        variableRateSlope2: EngineFlags.KEEP_CURRENT,
+        stableRateSlope1: EngineFlags.KEEP_CURRENT,
+        stableRateSlope2: EngineFlags.KEEP_CURRENT,
+        baseStableRateOffset: EngineFlags.KEEP_CURRENT,
+        stableRateExcessOffset: EngineFlags.KEEP_CURRENT,
+        optimalStableToTotalDebtRatio: EngineFlags.KEEP_CURRENT
+      })
+    });
+    rateStrategies[1] = IAaveV3ConfigEngine.RateStrategyUpdate({
+      asset: AaveV3ArbitrumAssets.USDC_UNDERLYING,
+      params: IV3RateStrategyFactory.RateStrategyParams({
+        optimalUsageRatio: EngineFlags.KEEP_CURRENT,
```

Comment:
do we not put uOptimal to 90% because its not the native USDC?

Reply:
it was actualy a bit ambigious wether bridged usdc also qualified or not for Uopt increase, so picked the less risky option